### DISCUSSION TITLE
With limited internet access till 1st August

### DISCUSSION THREAD

**Discussion by sansyrox** [sparckles/Robyn#50] [Announcements]
Hey Everyone! 👋 

I will starting my first job soon and will be moving away from my family for the first time in 22 years(my age jic)!! So, I have decided to spend some quality time with the family and I will be away for the the next 15 days without/with limited internet access. 

I really appreciate all your contributions and am delighted to work with you all ❤️ However, I won't be very active till 1st August. 

I will still have some limited access, so, if anything urgent comes up , you can send me an email with "URGENT" (all caps) in the subject line or you can open an issue here with "URGENT" in the title.

I hope y'all understand this. Can't wait to be back working on this amazing project. 🔥 

May the FOSS be with you! 🖖

**Comment by shivaylamba** [comment]
Enjoy your time @sansyrox, the community members will also help you out!
